manifests/w/WavesAudioLtd/WavesCentral/12.0.20/WavesAudioLtd.WavesCentral.installer.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: WavesAudioLtd.WavesCentral

```

```suggestion
PackageIdentifier: WavesAudio.WavesCentral
```
You also need to edit folder name for this.